deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -33,7 +31,8 @@ spec:
         storage:
           strategy: persistent
           persistent:
-            pvcStorageRequest: 20Gi
+            storageSelector: {}
+            pvcStorageRequest: 20G
```

Comment:
Noticed this size inconsistency. I don't know of any storage system that will automatically re-provision to a smaller size once a PVC is bound, but if there are concerns about this change I could drop it.